client/src/components/header.js

```diff
@@ -20,7 +20,11 @@ export const Header = () => {
     <div className="headerContainer">
       <div className={path === '/' ? '' : 'headerBackGround'}>
         <div className="header">
+<<<<<<< HEAD
```

Comment:
Git puts these markers to divide branches in a merge conflict, so you'll want to remove them (and lines 25 and 27).

Reply:
Forgot to save that last file. Fixed now.